### PR TITLE
f-tabs@0.9.0 - Make tab title reactive

### PR DESCRIPTION
**Pull request by xander-marjoram**
### Added
- `UPDATE_TITLE` injection to make tab title reactive.

---

## UI Review Checks
- [ ] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]
- [ ] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

File: packages/components/molecules/f-tabs/src/components/tests/Tab.test.js [hunk 1]
```diff
@@ -14,25 +15,29 @@ const REGISTER_DATA = {
...
+        animationDirection: 'LEFT',
+        animate: true
+    },
+    [UPDATE_TITLE]: updateTitleMock
+};
+
 describe('Tab.vue', () => {
     let activeWrapper;
 
     function defineActiveComponent () {
```
Comment (minrwhite) [inline packages/components/molecules/f-tabs/src/components/tests/Tab.test.js:34]: Nice tidying up in here and below 👏🏽👏🏽👏🏽

**Review by minrwhite — DISMISSED** [review]

File: packages/components/molecules/f-tabs/stories/Tabs.stories.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ export const VueTabsComponent = (args, { argTypes }) => ({
 VueTabsComponent.storyName = 'f-tabs';
 VueTabsComponent.args = {
     selected: 'a',
-    tabTitle: 'I can be renamed'
+    tabTitle: 'How it works'
```
Comment (minrwhite) [inline packages/components/molecules/f-tabs/stories/Tabs.stories.js]: As the assertion is an `.includes()` this should work with:
```
    tabTitle: 'How it works (I can be renamed)';
```
If you want to keep the storybook sugar.

Comment (xander-marjoram) [reply packages/components/molecules/f-tabs/stories/Tabs.stories.js]: Good suggestion, I like it.

**Review by minrwhite — APPROVED** [review]
✨✨✨

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**